deltachat-rpc-client/tests/test_something.py

```diff
@@ -216,6 +217,31 @@ async def test_message(acfactory) -> None:
     await message.send_reaction("😎")
 
 
+@pytest.mark.asyncio
+async def test_is_bot(acfactory) -> None:
+    """Test that we can recognize messages submitted by bots."""
+    alice, bob = await acfactory.get_online_accounts(2)
+
+    bob_addr = await bob.get_config("addr")
+    alice_contact_bob = await alice.create_contact(bob_addr, "Bob")
+    alice_chat_bob = await alice_contact_bob.create_chat()
+    await alice_chat_bob.send_text("Hello!")
+
+    # Alice becomes a bot.
+    await alice.set_config("bot")
```

Comment:
```suggestion
    await alice.set_config("bot", "1")
```

Reply:
Alice becomes a bot after sending the message, shouldn't this happen before sending the message? I think the code below receiving the message and checking `is_bot` will fail